src/libponyrt/sched/scheduler.c

```diff
@@ -1293,7 +1292,7 @@ void ponyint_sched_mute(pony_ctx_t* ctx, pony_actor_t* sender, pony_actor_t* rec
     // This is safe because an actor can only ever be in a single scheduler's
     // mutemap
     ponyint_muteset_putindex(&mref->value, sender, index2);
-    uint64_t muted = atomic_load_explicit(&sender->muted, memory_order_relaxed);
+    size_t muted = atomic_load_explicit(&sender->muted, memory_order_relaxed);
```

Comment:
why the change to size_t?

Reply:
Good question. I had a reason once upon a time but can't recall now. I can revert it back if you prefer.

Reply:
I'm not sure what the impact would be on 32 bit platforms. If how we track number of actors is a size_t elsewhere that should be fine. This should match up with that. I seem to remember checking that previously and it was a uint64_t but I'm not sure. If you could confirm which it is and make it match that would be best. I'm not comfortable merging this bit of change without verifying that.

Reply:
We don't explicitly track number of actors anywhere (that I'm aware of). The closest we come is via the the cycle detector `views` viewmap where all actors get added when they are created and the runtime hashmap internally uses `size_t` for `count` and `size`.